### PR TITLE
Feat/ allow users to remain in sim until they have free collateral in their vault.

### PR DESCRIPTION
**Pull request by Brendan-Lucas**
# Description
Allow users to remain in simulation mode so long as they have no free collateral added in perp. 

## Sanity Check
connect a wallet with no amount in vault. 
you should see that you are still in sim mode. 
Screen Shot 2022-10-25 at 6 52 55 PM

### CODE REVIEW

**Review by mattiaslightstone — CHANGES_REQUESTED** [review]
Add a pill to indicate that we are in simulation mode

File: src/containers/AddLiquidityModule/LiquidityAmountInput.tsx [hunk 1]
```diff
@@ -35,15 +37,21 @@ export const LiquidityAmountInput = () => {
     const { instance: vaultInstance } = useVaultContract()
     const { chain } = useNetwork()
 
+    const [simulationMode, setSimulationMode] =
+        useRecoilState(simulationModeAtom)
+
     const [freeCollateral, setFreeCollateral] =
         useRecoilState(freeCollateralAtom)
 
+    const totalCollateral = useRecoilValue(totalCollateralAtom)
```
Comment (nickwang14) [inline src/containers/AddLiquidityModule/LiquidityAmountInput.tsx:46]: This looks unused

**Review by nickwang14 — APPROVED** [review]
Looks good

**Comment by nickwang14** [comment]
> Add a pill to indicate that we are in simulation mode

Done! 🎉

**Pull request merged**